01-messaging.md

```diff
@@ -331,24 +335,27 @@ The fundee node:
     - MUST use `temporary_channel_id` in lieu of `channel_id`.
 
 A sending node:
-  - when sending `error`:
-    - MUST fail the channel referred to by the error message.
   - SHOULD send `error` for protocol violations or internal errors that make channels unusable or that make further communication unusable.
   - SHOULD send `error` with the unknown `channel_id` in reply to messages of type `32`-`255` related to unknown channels.
+  - when sending `error`:
+    - MUST fail the channel(s) referred to by the error message.
```

Comment:
nit: referred to by the `error` ~~message~~ (pending on: #941 )

the message has only one `channel_id` field. BOLT 1 says if the  `id` is zero it refers to all channels between the peers. We ignore the situation where we have `n` channels and want to refer to `m` channels with 1 `error`?

Reply:
That'd likely require multiple warning messages, one for each channel. Though it's hard to imagine a scenario where m < n and m > 1 channels need to close due to the same issue. The all-channel error was mostly intended to "there's something wrong but I can't identify the channel it's affecting" afaik, so "there's something wrong with one of these m channels" isn't much more useful IMHO.